packages/app-project/src/screens/ProjectHomePage/components/Hero/components/Introduction/Introduction.js

```diff
@@ -0,0 +1,45 @@
+import { SpacedText } from '@zooniverse/react-components'
+import counterpart from 'counterpart'
+import { Anchor, Box, Text } from 'grommet'
+import { Next } from 'grommet-icons'
+import Link from 'next/link'
+import { object, string } from 'prop-types'
+import React from 'react'
+
+import en from './locales/en'
+
+counterpart.registerTranslations('en', en)
+
+function Introduction (props) {
+  const { description, link, title } = props
+  return (
+    <Box>
+      <Box margin={{ bottom: 'xsmall' }}>
+        <SpacedText weight='bold'>
+          {title}
+        </SpacedText>
+      </Box>
+      <Box margin={{ bottom: 'small' }}>
+        <Text size='xlarge' weight='bold'>
```

Comment:
Instead of a `Box` and `Text`, why not just `Paragraph` as that's a block element with margin already and also has semantic meaning?